### PR TITLE
kubelet: add RSS memory to the summary API

### PR DESCRIPTION
**Pull request by yujuhong**

### CODE REVIEW

**Comment by dchen1107** [comment]
LGTM

**Comment by dchen1107** [comment]
LGTM

File: pkg/kubelet/api/v1alpha1/stats/types.go [hunk 1]
```diff
@@ -133,6 +133,9 @@ type MemoryStats struct {
 	// The amount of working set memory. This includes recently accessed memory,
 	// dirty memory, and kernel memory. UsageBytes is <= TotalBytes.
 	WorkingSetBytes *uint64 `json:"workingSetBytes,omitempty"`
+	// The amount of anonymous and swap cache memory (includes transparent
+	// hugepages).
+	RSSBytes *uint64 `json:"rssBytes, omitempty"`
```
Comment (timstclair) [inline pkg/kubelet/api/v1alpha1/stats/types.go]: (nit) no space after `,` (not sure if it matters, but it's always written like this)

Comment (yujuhong) [inline pkg/kubelet/api/v1alpha1/stats/types.go]: that was not intentional. will fix it.

**Comment by yujuhong** [comment]
Removed the extra space and reapplied the ligtm label.

**Comment by dchen1107** [comment]
cc/ @roberthbailey or @bgrant0607 can one of you add cherrypick-approval label for this pr?

**Comment by roberthbailey** [comment]
@zmerlynn and I will go through the list of pending cherry picks tomorrow.

**Comment by roberthbailey** [comment]
@dchen1107 cherry pick approved -- can you or @yujuhong create a cherry pick pull?

**Pull request merged**